flocker/provision/_install.py

```diff
@@ -114,6 +114,37 @@ def task_upgrade_kernel():
     ]
 
 
+KOJI_URL_TEMPLATE = (
+    'https://kojipkgs.fedoraproject.org/packages/kernel'
+    '/{version}/{release}.{distribution}/{architecture}'
+    '/kernel-{version}-{release}.{distribution}.{architecture}.rpm'
+)
+
+
+def koji_kernel_url(version, release, distribution, architecture):
+    """
+    Return the koji URL for the given kernel version.
+    """
+    url = KOJI_URL_TEMPLATE.format(
+        version=version,
+        release=release,
+        distribution=distribution,
+        architecture=architecture
+    )
+    return url
+
+
+def task_install_kernel(version='3.16.6', release='203', distribution='fc20',
```

Comment:
Ah the defaults seem to be used in `.. task:: install_kernel` in the docs. It might be sensible to note that here. Also, as noted elsewhere, the defaults are now out of date, or at least they don't specify the latest version. It might be too much to find the latest version at run time, but if not, do that. Else update them so they're at least up to date now.

How bad is it to specify an old kernel version here?